src/app/app.component.ts

```diff
@@ -41,4 +44,22 @@ export class AppComponent {
 
     });
   }
+
+  ngOnInit(): void {
+    const appTitle = this.titleService.getTitle();
```

Comment:
It might be cleaner to bake this functionality into the subscription that is already in the constructor as that is already doing an action on navigation end.  It also responds to ChildActivationEnd, which you will need to too as there as different titles for the quick maps sub-pages.  This means you will need to set route data.title attributes on the quickmaps sub-routes too:
https://github.com/micronutrientsupport/micronutrient-support-tool/blob/develop/src/app/pages/quickMaps/quickMaps-routing.module.ts

Reply:
was just looking into how the quick maps routes are wired up. I will try move the code into the constructor, makes a lot more sense there :)